conf/application.conf

```diff
@@ -50,6 +50,9 @@ play.http.errorHandler = "uk.gov.hmrc.plasticpackagingtax.registration.config.Er
 
 play.filters.csp.CSPFilter = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com www.googletagmanager.com tagmanager.google.com data:"
 
+# Inject a default system clock where needed
+play.modules.enabled += "uk.gov.hmrc.plasticpackagingtax.registration.config.Module"
+
```

Comment:
This is already in the file, a few lines above.

Reply:
Removed the duplicate line